core/modules/simpletest/tests/form_test.module

```diff
@@ -2485,6 +2492,25 @@ function _form_test_checkboxes_zero_no_redirect($form, &$form_state) {
   $form_state['redirect'] = FALSE;
 }
 
+/**
+ * Menu callback for form-test/textarea-resize.
+ */
+function form_test_resizable_textarea($form, &$form_state) {
+  // Use the old boolean value.
+  $form['text'] = array(
+    '#type' => 'textarea',
+    '#title' => 'Test',
+    '#resizable' => TRUE,
+  );
```

Comment:
Could you validate each option? `TRUE/FALSE/'vertical'/'horizontal'/'none'` and some invalid value like `'true'`.

Reply:
The test checks for more values now.